ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/nodes/pairs/values/TomlLiteralString.kt

```diff
@@ -92,8 +88,25 @@ public class TomlLiteralString internal constructor(
          */
         private fun String.convertSingleQuotes(): String = this.replace("\\'", "'")
 
-        private fun String.escapeQuotesAndVerify(config: TomlOutputConfig) =
+        private fun String.escapeQuotesAndVerify(config: TomlOutputConfig, multiline: Boolean) =
                 when {
+                    multiline ->
+                        when {
+                            multilineControlCharacterRegex in this ->
```

Comment:
Regular expressions affect performance and in some cases can lead to exceptions in case of very long strings...
May it will be better to use something like `string.any(setOfillegalCharacters::contains)` instead?

Reply:
Sure, admittedly regexes aren't needed here. I assume you want a separate issue for this?

Reply:
Yeah, I will make a separate one